src/helpers/actions.ts

```diff
@@ -7,19 +7,21 @@ export async function addOrUpdateSpace(space: string, settings: any) {
 
   const ts = (Date.now() / 1e3).toFixed();
   const query =
-    'INSERT INTO spaces SET ? ON DUPLICATE KEY UPDATE updated = ?, settings = ?, name = ?, hibernated = 0';
+    'INSERT INTO spaces SET ? ON DUPLICATE KEY UPDATE updated = ?, settings = ?, name = ?, hibernated = 0, domain = ?';
 
   await db.queryAsync(query, [
     {
       id: space,
       name: settings.name,
       created: ts,
       updated: ts,
-      settings: JSON.stringify(settings)
+      settings: JSON.stringify(settings),
+      domain: settings.domain || null
```

Comment:
i think better to be undefined 
```suggestion
      domain: settings.domain || undefined
```

Reply:
What will it do on the SQL side when passing undefined, and when there is previous value ? Will it just skip this column, or set undefined ? 

This column is only accepting either NULL or string, so what will undefined be ?

Reply:
ahh i was assuming it to be `domain` inside JSON 😆